sphinx/json_ref.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+
+import importlib
+import json
+
+import jinja2
+
+from bokeh.plot_object import PlotObject, Viewable
+from bokeh.protocol import serialize_json
+
+ITEM = jinja2.Template("""
+.. code-block:: javascript
+
+    {{ model_json|indent(4) }}
+""")
+
+
+module_names = [
+    'bokeh.models.axes',
+    'bokeh.models.formatters',
+    'bokeh.models.glyphs',
+    'bokeh.models.grids',
+    'bokeh.models.map_plots',
+    'bokeh.models.mappers',
+    'bokeh.models.markers',
+    'bokeh.models.plots',
+    'bokeh.models.ranges',
+    'bokeh.models.renderers',
+    'bokeh.models.sources',
+    'bokeh.models.tickers',
+    'bokeh.models.tools',
+    'bokeh.models.widget',
+]
```

Comment:
Absolutely, in a follow on PR. This is just what I wrote in 10 minutes to get something in the release. I am going to make several issues and master issues today about improvements to docs build.